### DISCUSSION TITLE
OSError: [E053] Could not read config.cfg from /usr/local/lib/python3.8/dist-packages/fr_core_news_sm/fr_core_news_sm-2.2.5/config.cfg

### DISCUSSION THREAD

**Discussion by Ichraknaceur** [explosion/spaCy#11577] [Help: Installation] [answered]
## Your Environment

* Operating System: ubuntu 
* Python Version Used: 3.8
* spaCy Version Used: spaCy v2.2.4
* Environment Information:
i installed fr_core_news_sm==2.2.5
i tried  of changing versions but i always get the same error

**Comment by polm** [comment, marked as answer]
Sorry you're having trouble with this.

spaCy v2.2.4 is over two years old at this point, is there a reason you're using that particular version? If so, it'd help to know that so we can understand other things about your environment.

Can you share the full text of the error you get? I've seen errors like that when trying to load v2 models from v3 sometimes, but a stack trace would be helpful for figuring out exactly what's going on. You can confirm what spaCy version you have installed with `spacy info`.

**Reply by Ichraknaceur** [reply]
there is no reason to use this version , i just want a version which is compatible with the model fr_core_news_sm

**Reply by Ichraknaceur** [reply]
```
---------------------------------------------------------------------------
OSError                                   Traceback (most recent call last)
Cell In [62], line 1
----> 1 nlp = spacy.load('fr_core_news_sm')
      2 data = ResumeParser('CV_Ennaceur_Ichrak.pdf').get_extracted_data()

File /usr/local/lib/python3.8/dist-packages/spacy/__init__.py:47, in load(name, disable, exclude, config)
     29 def load(
     30     name: Union[str, Path],
     31     disable: Iterable[str] = util.SimpleFrozenList(),
     32     exclude: Iterable[str] = util.SimpleFrozenList(),
     33     config: Union[Dict[str, Any], Config] = util.SimpleFrozenDict(),
     34 ) -> Language:
     35     """Load a spaCy model from an installed package or a local path.
     36 
     37     name (str): Package name or model path.
   (...)
     45     RETURNS (Language): The loaded nlp object.
     46     """
---> 47     return util.load_model(name, disable=disable, exclude=exclude, config=config)

File /usr/local/lib/python3.8/dist-packages/spacy/util.py:322, in load_model(name, vocab, disable, exclude, config)
    320 # https://stackoverflow.com/a/39662359/6400719
    321 try:
--> 322     shell = get_ipython().__class__.__name__
    323     if shell == "ZMQInteractiveShell":
    324         return True  # Jupyter notebook or qtconsole

File /usr/local/lib/python3.8/dist-packages/spacy/util.py:355, in load_model_from_package(name, vocab, disable, exclude, config)
    353 array = cupy.ndarray(numpy_array.shape, order="C", dtype=numpy_array.dtype)
    354 array.set(numpy_array, stream=stream)
--> 355 return array

File /usr/local/lib/python3.8/dist-packages/fr_core_news_sm/__init__.py:12, in load(**overrides)
     11 def load(**overrides):
---> 12     return load_model_from_init_py(__file__, **overrides)

File /usr/local/lib/python3.8/dist-packages/spacy/util.py:514, in load_model_from_init_py(init_file, vocab, disable, exclude, config)
    512 else:
    513     size_ = size
--> 514 items = iter(items)
    515 while True:
    516     batch_size = next(size_)

File /usr/local/lib/python3.8/dist-packages/spacy/util.py:388, in load_model_from_path(model_path, meta, vocab, disable, exclude, config)
    386     meta = get_model_meta(model_path)
    387 config_path = model_path / "config.cfg"
--> 388 config = load_config(config_path, overrides=dict_to_dot(config))
    389 nlp = load_model_from_config(config, vocab=vocab, disable=disable, exclude=exclude)
    390 return nlp.from_disk(model_path, exclude=exclude)

File /usr/local/lib/python3.8/dist-packages/spacy/util.py:545, in load_config(path, overrides, interpolate)
    543 else:
    544     if not config_path or not config_path.exists() or not config_path.is_file():
--> 545         raise IOError(Errors.E053.format(path=config_path, name="config.cfg"))
    546     return config.from_disk(
    547         config_path, overrides=overrides, interpolate=interpolate
    548     )

OSError: [E053] Could not read config.cfg from /usr/local/lib/python3.8/dist-packages/fr_core_news_sm/fr_core_news_sm-2.2.5/config.cfg
​```

**Reply by polm** [reply]
> there is no reason to use this version , i just want a version which is compatible with the model fr_core_news_sm

If that is the case I would strongly recommend you upgrade to spaCy v3, such as by using `pip install -U spacy`. The small french pipeline is still supported.

Can you confirm what version of spaCy you have? You can check this with `spacy info` or `pip freeze | grep spacy`. It looks like you have v3 installed already, and should just be able to install a compatible pipeline following the [quickstart guide](https://spacy.io/usage).

**Reply by Ichraknaceur** [reply]
Thank u for ur answer .
it worked for me by changing the scpacy version to 2.3.5 
and i use the models : 
TYPE      NAME              MODEL             VERSION                            
package   en-core-web-sm    en_core_web_sm    2.3.1   ✔
package   fr-core-news-sm   fr_core_news_sm   2.3.0   ✔

**Reply by polm** [reply]
I'm glad you got it working, but that version of spaCy and those models are pretty old. If you're just getting started, I would strongly suggest going through the quickstart I linked to get the latest versions.
